driver/userspace.c

```diff
@@ -762,7 +762,7 @@ WnbdParseUserIOCTL(PWNBD_EXTENSION DeviceExtension,
             DeviceExtension, ReqCmd->ConnectionId, TRUE);
         if (!Device) {
             Status = STATUS_INVALID_HANDLE;
-            WNBD_LOG_WARN(
+            WNBD_LOG_INFO(
```

Comment:
I'd rather use a debug message since there can be hundreds of timed out requests.